src/fees/saga.ts

```diff
@@ -161,11 +163,16 @@ export function* estimateSwapFee(tokenAddress: string) {
   // TODO: calculate the fee accurately.
   // To calculate the fee, you need the txo but in the in-app swaps flow the
   // swap amount is required to query the 0x API for the txo. For now,
-  // approximate the fee to be 4x that of a simple transfer, to take into
-  // account long swap routes. The maximum i saw on Ubeswap was 3, choosing 4 as
-  // a buffer. The tradeoff for this approximation is that users may have more
-  // dust (on the order of 3x the gas fee)
-  const feeInfo: FeeInfo = yield call(calculateFeeForTx, tx.txo, SWAP_FEE_ESTIMATE_MULTIPLIER)
+  // approximate the fee to be 8x that of a simple transfer, to take into
+  // account long swap routes.
+  // Increased multiplier for CELO swaps because the ratio swap_fee / simple_transaction_fee is higher
```

Comment:
I don't fully follow the explanation for CELO, why do we need a bigger multiplier for CELO?

Reply:
Yeah, sorry, I didn't know how to explain this.
But basically, in our calculations, all non-CELO transaction fees are much higher than the CELO transaction fee. So, using the same multiplier for CELO is not reaching the fee needed for the swaps. But it does work for non-CELO tokens.

Imaginary example:

Transaction fee in CELO: 0.0001
Transaction fee in cUSD: 0.001

Minimum swap fee needed in CELO: 0.01
Minimum swap fee needed in cUSD: 0.01

So using the same multiplier is won't work here. I mean, we can use the CELO multiplier for everything, but we will leave more dust for non-CELO tokens.